tests/br_systables/run.sh

```diff
@@ -37,7 +37,8 @@ check() {
     run_sql "SELECT count(*) from mysql.foo;" | grep 11
     run_sql "SELECT count(*) from mysql.usertable;" | grep 1000
     run_sql "SHOW TABLES IN mysql;" | grep -v bar
```

Comment:
```suggestion
    run_sql "SHOW TABLES IN mysql;" | awk '/bar/{exit 1}'
```

According to the log, the table `mysql.bar` has been restored, but there are other tables too, so `grep` generated a false negative result. 🤔

(the log: `[2021-05-13T06:53:25.176Z] [2021/05/13 14:53:24.980 +08:00] [INFO] [systable_restore.go:148] ["successfully restore system database"] [table=bar] [database=mysql] [sql="RENAME TABLE `__tidb_br_temporary_mysql`.`bar` TO `mysql`.`bar`;"]`, from [here](https://ci.pingcap.net/blue/rest/organizations/jenkins/pipelines/br_ghpr_unit_and_integration_test/runs/1797/nodes/74/steps/1073/log/?start=0))